### DISCUSSION TITLE
The fault in time domain simulation

### DISCUSSION THREAD

**Discussion by Tang115** [CURENT/andes#232] [General]
Dr. Cui,

I like to know if the post-fault topology is the same as the pre-fault topology in the assumption of Andes?

If not, is there any way to switch the topology back (reclose the breaker) after the fault is cleared?

Thanks!

Best regard

**Comment by cuihantao** [comment]
> I like to know if the post-fault topology is the same as the pre-fault topology in the assumption of Andes?

A `Fault` is essentially a very small reactance shunt to ground. It does not affect system topology.

> is there any way to switch the topology back (reclose the breaker) after the fault is cleared?

`Toggler` changes the on/off status of elements. If a toggler is used to change an online `Line` to offline, you can add another toggler and apply it to the same line to toggle it back online, which restores the topology.
